pkg/oc/cli/cmd/startbuild_test.go

```diff
@@ -79,30 +79,6 @@ func TestStartBuildWebHook(t *testing.T) {
 	}
 }
 
-func TestStartBuildWebHookHTTPS(t *testing.T) {
-	invoked := make(chan struct{}, 1)
-	server := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		invoked <- struct{}{}
-		w.WriteHeader(http.StatusOK)
-	}))
-	defer server.Close()
-
-	testErr := errors.New("not enabled")
-	cfg := &FakeClientConfig{
-		Err: testErr,
-	}
-	buf := &bytes.Buffer{}
-	o := &StartBuildOptions{
-		Out:          buf,
-		ClientConfig: cfg,
-		FromWebhook:  server.URL + "/webhook",
-		Mapper:       legacyscheme.Registry.RESTMapper(),
-	}
-	if err := o.Run(); err == nil || !strings.Contains(err.Error(), "certificate signed by unknown authority") {
```

Comment:
This test never actually tested what it claimed to test.  It mocked an error from the wrong point to indicate a failure during actual communication.  That hook point no longer exists since this was actually a failure during client construction.

Reply:
@bparees fyi, your team will need to update it to do what it should be doing.

Reply:
i don't think this test ever had any real value (even if it had been implemented correctly).  I'm fine w/ losing it.